mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -4570,21 +4431,13 @@ LogicalResult OpenMPDialectLLVMIRTranslationInterface::convertOperation(
     Operation *op, llvm::IRBuilderBase &builder,
     LLVM::ModuleTranslation &moduleTranslation) const {
 
-  // Check to see if there is a lowering that overrides the default lowering
-  // if not use the default dispatch.
-  bool match = false;
-  LogicalResult result = success();
-  std::tie(match, result) =
-      dispatchList.convertOperation(op, builder, moduleTranslation);
-  if (match)
-    return result;
-
   llvm::OpenMPIRBuilder *ompBuilder = moduleTranslation.getOpenMPBuilder();
   if (ompBuilder->Config.isTargetDevice()) {
-    if (isTargetDeviceOp(op))
-      return convertTargetDeviceOp(op, builder, moduleTranslation,
-                                   dispatchList);
-    return convertTargetOpsInNest(op, builder, moduleTranslation);
+    if (isTargetDeviceOp(op)) {
```

Comment:
Nit: I think the previous formatting is the preferred one (no unnecessary braces and no 'else' after 'return').